perun-core/src/main/java/cz/metacentrum/perun/core/implApi/FacilitiesManagerImplApi.java

```diff
@@ -426,6 +426,7 @@
 
 	/**
 	 * Get all facilities where the user is admin.
+	 * Including facilities from authorized groups where the user is a member.
```

Comment:
I would prefer something like "Including facilities, where user is a member of authorized group."